get_hashsets.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+require("dotenv").load();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const HIBP_API_ROOT = process.env.HIBP_API_ROOT;
+const HIBP_API_TOKEN = process.env.HIBP_API_TOKEN || '';
+const HIBP_AUTH = 'Bearer ' + new Buffer(HIBP_API_TOKEN);
+
+
+function getBreachHashset(breach) {
+  if (breach.IsActive && breach.IsVerified && breach.DataClasses.indexOf("Email addresses") > -1) {
```

Comment:
If `breach.DataClasses` is an array, you can use [`Array.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) instead of checking `.indexOf() > -1`.